Code/Framework/AzFramework/AzFramework/Application/Application.cpp

```diff
@@ -369,6 +370,7 @@ namespace AzFramework
             azrtti_typeid<AzFramework::SpawnableSystemComponent>(),
             azrtti_typeid<Physics::MaterialSystemComponent>(),
             AZ::Uuid("{624a7be2-3c7e-4119-aee2-1db2bdb6cc89}"), // ScriptDebugAgent
+            azrtti_typeid<AzFramework::QualitySystemComponent>(),
```

Comment:
Can you add this component to the GetRequiredComponents function in the `AzFrameworkModule.cpp` file?

Reply:
Yeah, that should work fine,  not sure why we have them split up this way - to ensure loading order?

Reply:
The only reason it is currently split is because the `AzCoreModule/AzFrameworkModule/AzToolsFrameworkModule` static module classes came later.

All of these required system components in the `Application::GetRequiredSystemComponents` function can be moved to `AzFrameworkModule`.

It provides a better separation of responsibilities as well as the Application wouldn't be responsible for specifying required System components.

Reply:
I'm currently testing moving these to `AzFrameworkModule` .

Reply:
Moving all system components to the module results in a few blocking issues, some are simple, like removing `TaskGraphSystemComponent` which is already in the `AzCoreModule` and am currently trying to sort out a  blocking dependency issue with AssetSystemComponent - it looks like the `Application::GetRequiredSystemComponents` was executed before the system entity was activated, and module manager is added the other system components later, which is currently what I'm looking at.

Reply:
You don't have to move them all if they are causing you issues, since that is not directly related to this PR.

Reply:
I think I will hold off on moving these because I need to also move the ones for all the other Applications sub classes that are using `GetRequiredSystemComponents` which have dependencies on some of these which could take a bit to sort out.